### DISCUSSION TITLE
discord-screenaudio stopped working

### DISCUSSION THREAD

**Discussion by xxhinotorixx** [maltejur/discord-screenaudio#201] [Q&A] [answered]
Hi everyone!

Has discord-screenaudio stopped working for anyone else?
Since a day or two the GUI seems not to be loaded correctly.
When I run discord-screenaudio through the terminal, it returns the following:

`[discord] [GatewaySocket] Setting connection state to SESSION_ESTABLISHED
[discord] [Flux] Dispatching CONNECTION_OPEN
[discord] [AuthenticationStore] handleConnectionOpen called [object Object]
[discord] [DatabaseManager] removing database (user: 339498780498067477, database: null)
[discord] [DatabaseManager] added database (339498780498067477 → null)
[discord] [GatewaySocket] Updating resume url to wss://gateway-us-east1-d.discord.gg
[discord] [GatewaySocket] resetSocketOnError TypeError: Object.hasOwn is not a function
[discord]     at Function.asBigFlag (https://discord.com/assets/2d60acee3dd6d8a63c76.js:1:998560)
[discord]     at j.canBasicChannel (https://discord.com/assets/2d60acee3dd6d8a63c76.js:1:824737)
[discord]     at https://discord.com/assets/b8f7ac37ee6bd14069da.js:1:9850
[discord]     at z (https://discord.com/assets/b8f7ac37ee6bd14069da.js:1:10122)
[discord]     at X (https://discord.com/assets/b8f7ac37ee6bd14069da.js:1:11413)
[discord]     at https://discord.com/assets/e8fd2221603a9472ac4a.js:1:900919
[discord]     at https://discord.com/assets/e8fd2221603a9472ac4a.js:1:907154
[discord]     at m._dispatch (https://discord.com/assets/e8fd2221603a9472ac4a.js:1:900911)
[discord]     at https://discord.com/assets/e8fd2221603a9472ac4a.js:1:900481
[discord]     at s.log (https://discord.com/assets/e8fd2221603a9472ac4a.js:1:907087)
[discord] [GatewaySocket] [RESET] (true, 1000, Resetting socket due to error.)
[discord] [Flux] Dispatching CONNECTION_CLOSED
[discord] [AuthenticationStore] handleConnectionClosed called with code 1000. [object Object]
[discord] [ConnectionStore] connection closed dispatched
[discord] [GatewaySocket] Setting connection state to WILL_RECONNECT
[userscript] Cannot locate Mute/Deafen/Settings button container
[userscript] Cannot locate Mute/Deafen/Settings button container
[userscript] Cannot locate Mute/Deafen/Settings button container
[userscript] Cannot locate Mute/Deafen/Settings button container
[userscript] Cannot locate Mute/Deafen/Settings button container
[userscript] Cannot locate Mute/Deafen/Settings button container
[userscript] Cannot locate Mute/Deafen/Settings button container
[userscript] Cannot locate Mute/Deafen/Settings button container
[userscript] Cannot locate Mute/Deafen/Settings button container
[userscript] Cannot locate Mute/Deafen/Settings button container
[userscript] Cannot locate Mute/Deafen/Settings button container
[userscript] Cannot locate Mute/Deafen/Settings button container
[userscript] Cannot locate Mute/Deafen/Settings button container
[userscript] Cannot locate Mute/Deafen/Settings button container
[userscript] Cannot locate Mute/Deafen/Settings button container
[userscript] Cannot locate Mute/Deafen/Settings button container
[userscript] Cannot locate Mute/Deafen/Settings button container
[userscript] Cannot locate Mute/Deafen/Settings button container
[userscript] Cannot locate Mute/Deafen/Settings button container
[userscript] Cannot locate Mute/Deafen/Settings button container
`
I tried to reinstall the package via AUR and even tried to compile it myself from the source, same result.

Thanks in advance!

**Comment by AntonioFLB** [comment]
Yup, I can confirm, that it stopped working properly within the last couple weeks.
When I click on the screen sharing icon, instead of getting the dialog to choose resolution, fps and audio source, it immediately starts sharing the one display, but no sound.

**Reply by maltejur** [reply, marked as answer]
Did you update to the latest version?

**Reply by AntonioFLB** [reply]
```
Information for package discord-screenaudio:
--------------------------------------------
Repository     : @System
Name           : discord-screenaudio
Version        : 1.8.0-15.5
Arch           : x86_64
Vendor         : obs://build.opensuse.org/games:tools
Installed Size : 1,7 MiB
Installed      : Yes
Status         : up-to-date
Source package : discord-screenaudio-1.8.0-15.5.src
Upstream URL   : https://github.com/maltejur/discord-screenaudio
Summary        : A very WIP custom discord client that supports streaming with audio on Linux
Description    :
A very WIP custom discord client that supports streaming with audio on Linux, made possible by the great work of
edisionnano and the Rohrkabel library by Curve.
```
~~That's the information I'm getting on my openSUSE system. So, I don't have the latest verison (v1.9.2), but I don't know how to get or build the latest version either. ~~

Correction: I just got the latest version though [Flathub](https://flathub.org/apps/de.shorsh.discord-screenaudio) and it works. Thanks!
